nibabel/nicom/utils.py

```diff
@@ -51,3 +53,138 @@ def find_private_section(dcm_data, group_no, creator):
             if creator == name:
                 return elno * 0x100
     return None
+
+
+def tm_to_seconds(time_str):
+    '''Convert a DICOM time value (value representation of 'TM') to the number
```

Comment:
Needs to be single descriptive line - maybe `Convert DICOM time value (VR of 'TM') to seconds past midnight` ?